tests/playback.js

```diff
@@ -117,6 +129,16 @@ queue_next_packet = function () {
 var bytes_processed = 0;
 
 do_packet = function () {
+    // Avoid having an excessive queue buildup
+    if (rfb._flushing) {
```

Comment:
Yep, woke up in the middle of the night last night and realized what you were saying :-P.  I'd still prefer if we could only turn it on "performance" mode though, since I use recordings to debug issues in the realitime mode, and sometimes timing matters there.

Honestly, though, sometimes checking buffer management is what I want to do with the performance test, though.  Could we maybe just have it behind an option (I've used this code to fix GC-related performance issues in our buffer management, for instance)?